cli_tools/common/utils/storage/storage_client.go

```diff
@@ -31,7 +31,9 @@ import (
 )
 
 var (
-	gsRegex = regexp.MustCompile(`^gs://([a-z0-9][-_.a-z0-9]*)/(.+)$`)
+	bucket = `([a-z0-9][-_.a-z0-9]*)`
+	bucketRegex = regexp.MustCompile(fmt.Sprintf(`^gs://%s/?$`, bucket))
+	gsRegex = regexp.MustCompile(fmt.Sprintf(`^gs://%s/(.+)$`, bucket))
```

Comment:
the variable names are not entirely clear.
"bucket" is essentially the regex that follows gcs naming pattern, right? if so, it should be changed to bucketregex. and then bucketregex, gsregex should be changed to something more specific to what pattern the regex is supposed to match.

Reply:
Sounds good, however I followed pattern just like in daisy/storage.go to keep the style. We may have a bigger refactoring to resolve this naming confusion.

Reply:
We can do it step by step. Readability trumps consistency IMO.